composer.json

```diff
@@ -22,6 +22,7 @@
     "easyframework/collections": "dev-master"
   },
   "require-dev": {
+    "php-amqplib/php-amqplib": "2.6.*",
```

Comment:
Not really, Dev packages are installed by default https://getcomposer.org/doc/03-cli.md, just search for *--dev*, unless you use the *--no-dev* flag to not install required dev packages. Once we have more then 5 adapters then we need to add all of them to dev? What if I don't want to install kafka package?